src/OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -82,6 +82,10 @@ void CaptureEventProcessor::ProcessEvent(const ClientCaptureEvent& event) {
     case ClientCaptureEvent::kSystemMemoryUsage:
       // TODO (http://b/179000848): Process the system memory usage information.
       break;
+    case ClientCaptureEvent::kApiEvent: {
```

Comment:
`CaptureEventProcessorTest.cpp` should probably be expanded accordingly. Refer to Florian and Dimitry, they worked on it the most.

Reply:
Added ApiEventProcessorTest.cpp which tests the processing of api events for both CaptureEventProcessor and ApiEventProcessor.

Reply:
See the other comment: there should be a small test regarding `CaptureEventProcessor` with `kApiEvent` in `CaptureEventProcessorTest.cpp`.